src/data_handlers/user_data_handler.js

```diff
@@ -1,62 +1,28 @@
 const axios = require('axios').default
 
-/**
- *
- * A class for interracting with user data
- * @class UserDataHandler
- */
 class UserDataHandler {
-  /**
-   *Creates an instance of UserDataHandler.
-   * @memberof UserDataHandler
-   */
   constructor () {
     this.users = []
   }
 
-  /**
-   * A method for loading users data from the server
-   * @return {Promise} if successful updates users array of the calass instance
-   * @memberof UserDataHandler
-   */
   async loadUsers () {
     const response = await axios.get('http://localhost:3000/users').catch(err => {
       throw new Error(`Failed to load users data: ${err}`)
     })
     this.users = response.data
```

Comment:
```suggestion
    this.users = await axios.get('http://localhost:3000/users')
      .then((response) => response.data)
      .catch(err => {
        throw new Error(`Failed to load users data: ${err}`)
      });
```

Reply:
thanks for the suggestion, but FYI I didn't touch user_data_handler.js file, it was in the repo I downloaded, I just wrote unit tests for it